### PR TITLE
Move run-tests.sh to a distinct folder

### PR DESCRIPTION
**Pull request by ppamorim**
Move this script to a separated folder to keep the project less clogged.

### CODE REVIEW

**Comment by ppamorim** [comment]
bors try

**Review by bidoubiwa — APPROVED** [review]

**Review by curquiza — CHANGES_REQUESTED** [review]
Could you update the CONTRIBUTING.md accordingly, please? 🙂

**Comment by ppamorim** [comment]
@curquiza Done, could you please review it again?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -40,14 +40,21 @@ swift build
...
 ```
 
-If you do not have a MeiliSearch instance with the previous mentioned parameters, you can run the test script that will automatically run a Docker container with MeiliSearch:
+If you do not have a MeiliSearch instance with the previous mentioned parameters, you can run the test script that will automatically run a Docker container with MeiliSearch and start the test:
+
+```bash
+./Scripts/run_test.sh
+```
+
+If you want to run the linter `swiftlint`:
```
Comment (curquiza) [inline CONTRIBUTING.md:54]:
```suggestion
If you want to run the linter [`swiftlint`](https://github.com/realm/SwiftLint):
```

**Review by curquiza — CHANGES_REQUESTED** [review]

**Comment by curquiza** [comment]
bors try

**Review by curquiza — APPROVED** [review]

**Comment by curquiza** [comment]
bors merge

**Pull request merged**